libs/crypto/keys_manager_impl.cpp

```diff
@@ -53,10 +53,10 @@ namespace iroha {
         iroha::sha3_256(test).to_string(), keypair.pubkey, sig);
   }
 
-  nonstd::optional<iroha::keypair_t> KeysManagerImpl::loadKeys() {
+  nonstd::optional<iroha::keypair_t> KeysManagerImpl::loadKeys(std::string pubpath, std::string privpath) {
```

Comment:
and as I see can be marked `const`

Reply:
keys manager will be removed soon, in next PR.